docs/Materials.md.html

```diff
@@ -932,6 +932,36 @@
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+### General: featureLevel
+
+Type
+:    `number`
+
+Value
+:     An integer value, either 1 or 2. Defaults to 1.
+
+      Feature Level    |        Guaranteed features
+:----------------------|:---------------------------------
+1                      | 9 textures per material
+2                      | 12 textures per material, cubemap arrays, ESSL 3.10
```

Comment:
How did you come up with the 9 and 12 texture limits?

Reply:
9 is what Impress is using now -- they use 9 and we're using 7. 
For feature level 2, I just picked 12 :-) I think we can revisit at some point.

Reply:
Okay, sounds good. Seems like if the backend supports 31 then we should be able to expose a lot more to materials.

Reply:
Well, we need to keep some for us. And we're already technically "in debt". We could do 15 (users) + 16 (us). I think it's a limit we can raise later anyways, it wouldn't cause incompatibilities.

Reply:
👍